src/main/js/store/actions/session.js

```diff
@@ -59,15 +59,12 @@ export function verifyLogin() {
         const csrfToken = getState().session.csrfToken;
```

Comment:
You can remove this line to get the csrf token since it's not being used in the request.